jbpm-human-task/jbpm-human-task-jpa/pom.xml

```diff
@@ -75,6 +75,11 @@
     </dependency>
 
     <!-- TEST -->
+    <dependency>
+      <groupId>org.jbpm</groupId>
+      <artifactId>jbpm-test-util</artifactId>
+      <scope>test</scope>
+    </dependency>
```

Comment:
that seems to be correct, there are no tests at all in that module so having dependencies and configuration for transaction does not make sense.